python/tvm/relax/dpl/pattern.py

```diff
@@ -1115,3 +1115,30 @@ def make_fused_bias_activation_pattern(op_name, with_bias=False, activation=None
         return is_op(activation)(out)
 
     return out
+
+
+def rewrite(
+    pattern: DFPattern, callback: Callable[[Expr, Dict[DFPattern, Expr]], Expr], func: Function
+) -> Function:
+    """
+    Rewrite a function with the given pattern and the callback.
+
+    Parameters
+    ----------
+    pattern: DFPattern
+        The pattern to match.
+
+    callback: Callable[[Expr, Dict[DFPattern, Expr]], Expr]
+        The function to be called on a successful matching for rewriting. Given the matching
+        call node and the map of patterns and matched expressions, it should return a new call node
+        or the original matched call node as is.
```

Comment:
Maybe explicitly saying "returns a new call node ... **to replace the original given call node**" would make things more clear.